forms-flow-bpm/src/main/resources/processes/telework-form.bpmn

```diff
@@ -73,40 +74,77 @@ task.execution.setVariable('deleteReason', "completed");</camunda:script>
         </camunda:connector>
       </bpmn:extensionElements>
       <bpmn:incoming>SequenceFlow_0ociprs</bpmn:incoming>
+    </bpmn:serviceTask>
+    <bpmn:sequenceFlow id="Flow_0eqfxon" sourceRef="StartEvent_1" targetRef="Activity_13ovu4t">
+      <bpmn:extensionElements>
+        <camunda:executionListener class="org.camunda.bpm.extension.hooks.listeners.FormBPMDataPipelineListener" event="take" />
+      </bpmn:extensionElements>
+    </bpmn:sequenceFlow>
+    <bpmn:serviceTask id="Activity_13ovu4t" name="Send Approval Email to Supervisor">
+      <bpmn:extensionElements>
+        <camunda:connector>
+          <camunda:inputOutput>
+            <camunda:inputParameter name="subject">FOR ACTION: A new Telework Agreement has been submitted</camunda:inputParameter>
+            <camunda:inputParameter name="to">${managerEmployerEMail}</camunda:inputParameter>
+            <camunda:inputParameter name="text">Hello
+
+
+For your review, please find attached a new Telework Agreement submitted by ${submitterName}.
+
+
+Go to the Manager/Supervisor Sign-Off Section at ${formUrl} to complete and submit this form.
```

Comment:
I think we might have to change this a bit, ideally the link would take you to the url of the task associated with it, and not the form itself 😅

Reply:
Oh yeah!  That makes sense.  This is proving to need a fair bit of rework.  The way we're currently sending the email, the task hasn't been created at that point, so we have no way of including the task id for reference.  Have to write a new java class listener to send an email when the user task is created.